### PR TITLE
A few fixes for nRF52840 feather QSPI and neopixel

### PR DESCRIPTION
**Pull request by tannewt**

### CODE REVIEW

**Comment by ladyada** [comment]
neopixel works!

File: ports/nrf/common-hal/microcontroller/Pin.c [hunk 1]
```diff
@@ -48,7 +48,6 @@ STATIC uint32_t claimed_pins[GPIO_COUNT];
 STATIC uint32_t never_reset_pins[GPIO_COUNT];
 
 void reset_all_pins(void) {
-    return;
```
Comment (ladyada) [inline ports/nrf/common-hal/microcontroller/Pin.c]: lol

**Comment by dhalbert** [comment]
Looks good to me, but `mpconfigboard.mk` is set to SPI, not QSPI filesystem? Does QSPI work now?

**Comment by dhalbert** [comment]
Merged anyway - I read Slack convo.

**Pull request merged**